lib/rucio/core/rule.py

```diff
@@ -2414,22 +2414,24 @@ def examine_rule(rule_id: str, *, session: "Session") -> dict[str, Any]:
 
 
 @transactional_session
-def get_evaluation_backlog(expiration_time: int = 600, *, session: "Session") -> tuple[int, datetime]:
+def get_evaluation_backlog(expiration_time: int = 600, *, session: "Session") -> Union[NoValue, tuple[int, datetime]]:
     """
     Counts the number of entries in the rule evaluation backlog.
     (Number of files to be evaluated)
 
     :returns:     Tuple (Count, Datetime of oldest entry)
     """
 
-    cached_backlog = REGION.get('rule_evaluation_backlog', expiration_time=expiration_time)
+    cached_backlog: Union[NoValue, tuple[int, datetime]] = REGION.get('rule_evaluation_backlog', expiration_time=expiration_time)
```

Comment:
This annotation is not necessary for the type checker, but I think it's okay to keep it

Reply:
I think we had to do something about it in the past because the type of `REGION.get()` is `Any`, but I can’t recall the specifics.